runtime/interpreter/value_accountcapabilitycontroller.go

```diff
@@ -33,9 +33,9 @@ type AccountCapabilityControllerValue struct {
 	BorrowType   ReferenceStaticType
 	CapabilityID UInt64Value
 
-	// tag is locally cached result of GetTag, and not stored.
-	// It is populated when the field `tag` is read.
-	tag *StringValue
+	// Tag is locally cached result of GetTag, and not stored.
+	// It is populated when the field `Tag` is read.
+	Tag *StringValue
```

Comment:
Does this need to be exported? It's just a cached value, so ideally it shouldn't be exported

Reply:
Currently it's exported because it needs to be settable in the `stdlib` module, specifically in `account.go`. The convention seemed to be that all the functions used by the actual controller values were defined in this file, but if we'd rather keep the tag private I can move the update function definition to this file.

Reply:
Ah, I see now. Yes, it might be nice to move the function over, so the Go `Tag` field can stay unexported